platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/protocol/SyncProtocol.java

```diff
@@ -173,19 +156,14 @@ public boolean shouldInitiate() {
             return false;
         }
 
-        // is there a reason to initiate?
-        if (!useCriticalQuorum || peerNeededForFallenBehind() || criticalQuorum.isInCriticalQuorum(peerId)) {
-            final boolean isLockAcquired = permitProvider.tryAcquire(peerId);
-
-            if (isLockAcquired) {
-                syncMetrics.updateSyncPermitsAvailable(permitProvider.getNumAvailable());
-                syncMetrics.outgoingSyncRequestSent();
-            }
+        final boolean isLockAcquired = permitProvider.tryAcquire(peerId);
```

Comment:
With this change, `peerNeededForFallenBehind` has no remaining usages

Reply:
removed